.eslintrc.js

```diff
@@ -57,7 +57,12 @@ module.exports = {
         }],
         'import/no-extraneous-dependencies': 'off',
         'import/no-unresolved': 'off',
-
+        'lines-between-class-members': [
```

Comment:
This looks like something we might want to add globally to [`eslint-config-fozzie`](https://github.com/justeat/eslint-config-fozzie) possibly?